From 3f0e8d98668bbda21dbe76b71bff77644a6d34b5 Mon Sep 17 00:00:00 2001
From: Ankit Jain <ankitja@vmware.com>
Date: Tue, 12 Apr 2022 14:21:37 +0000
Subject: [PATCH] sshd_config: Avoid duplicate entry

- Instead of appending the entry to the file,
  overwrite the entry which is already present
  in sshd_config
---
 sshd_config | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/sshd_config b/sshd_config
index c423eba..af8b702 100644
--- a/sshd_config
+++ b/sshd_config
@@ -29,9 +29,9 @@
 # Authentication:

 #LoginGraceTime 2m
-#PermitRootLogin prohibit-password
+PermitRootLogin no
 #StrictModes yes
-#MaxAuthTries 6
+MaxAuthTries 4
 #MaxSessions 10

 #PubkeyAuthentication yes
@@ -79,10 +79,10 @@ AuthorizedKeysFile	.ssh/authorized_keys
 # If you just want the PAM account and session checks to run without
 # PAM authentication, then enable this but set PasswordAuthentication
 # and KbdInteractiveAuthentication to 'no'.
-#UsePAM no
+UsePAM yes

-#AllowAgentForwarding yes
-#AllowTcpForwarding yes
+AllowAgentForwarding no
+AllowTcpForwarding no
 #GatewayPorts no
 #X11Forwarding no
 #X11DisplayOffset 10
@@ -90,11 +90,11 @@ AuthorizedKeysFile	.ssh/authorized_keys
 #PermitTTY yes
 #PrintMotd yes
 #PrintLastLog yes
-#TCPKeepAlive yes
+TCPKeepAlive no
 #PermitUserEnvironment no
-#Compression delayed
+Compression no
 #ClientAliveInterval 0
-#ClientAliveCountMax 3
+ClientAliveCountMax 2
 #UseDNS no
 #PidFile /var/run/sshd.pid
 #MaxStartups 10:30:100
--
2.23.1

